modules/programs/browserpass.nix

```diff
@@ -0,0 +1,90 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+with import ../lib/dag.nix;
+
+let
+  browsers = {
+    chrome = 1;
+    chromium = 2;
+    firefox = 3;
+    vivaldi = 4;
+  };
+in {
+  options = {
+    programs.browserpass = {
+      enable = mkEnableOption "the browserpass extension host application";
+
+      browsers = mkOption {
+        type = types.listOf types.str;
```

Comment:
This should be `type = types.listOf (types.enum (builtins.attrNames browsers));` to allow the man page to properly show the available options. It also will check the values so the assertion in the config section below is unnecessary.